server/api/users/users.test.ts

```diff
@@ -36,21 +37,108 @@ describe('auth router', () => {
     });
   });
 
-  describe('GET /', () => {
-    it('expects empty array if no users exist', async () => {
+  describe('GET /api/users', () => {
+    it('should have an API endpoint that exists', async () => {
       const res = await server.exec.get('/api/users/');
-      expect(res.body).toStrictEqual([]);
+
+      expect(res.status).toBe(200);
+    });
+
+    it('should return empty array if no users exist', async () => {
+      const res = await server.exec.get('/api/users/');
+
+      expect(res.body.users).toStrictEqual([]);
+    });
+
+    it('should return invalid page number with negative pages', async () => {
+      const res = await server.exec.get('/api/users/?page=-1');
+
+      expect(res.body.message).toEqual('Invalid page number');
+      expect(res.status).toBe(400);
```

Comment:
nit: status checks should always be first imo (i.e. check status before checking more specific body values)